.github/workflows/build.yml

```diff
@@ -17,7 +17,7 @@ jobs:
           fetch-depth: 0
       - uses: actions/setup-node@v2
         with:
-          node-version: 14.17.6
+          node-version: 16.14.0
```

Comment:
Was this change needed?

No issue for me to keep it in this PR, but I'm wondering since we're migrating to node 18 if it shouldn't be included here

Reply:
Actually you are right, I'll remove this from this PR, and let's focus on node versions later on, this PR is primarily about typescript 👌